src/Components/LandingPage/ExerciseListElement.tsx

```diff
@@ -1,4 +1,44 @@
-type Props = { exercise: string };
-export const ExerciseListElement = ({ exercise }: Props) => {
-  return <h1>{exercise}</h1>;
+import { ChartBar, Play } from 'phosphor-react';
+import { useNavigate } from 'react-router-dom';
+import { MouseEvent, useState } from 'react';
+
+const EXERCISE: string = 'exercise';
+
+type Props = { title: string; description: string };
+
+export const ExerciseListElement = ({ title, description }: Props) => {
+  let navigate = useNavigate();
+  const [exerciseID, setExerciseID] = useState('id');
+  const handleClick = (e: MouseEvent<HTMLElement>) => {
+    e.currentTarget.id === EXERCISE
+      ? navigate(`exercises/${exerciseID}`)
+      : navigate(`statistics/${exerciseID}`);
```

Comment:
Should we not only navigate to statistics if the target id is "statistics" or something like that?